### PR TITLE
2.8-hf-azure-diable-prop

### PR DESCRIPTION
**Pull request by resmaa**
**2.8 Hot Fix**

Disable properties request based on the cloud account flag

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/877#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/04-clusters/1-new-clusters/03-azure.md
Comment (codrin-iftimie) [inline content/docs/04-clusters/1-new-clusters/03-azure.md:90]: In the normal flow you'd still have to select those.
If you select an cloud account with Disable properties enabled then you have to input the right details as described: 
```
Name <br /> CIDR Block <br /> Security Group Name
```

File: content/docs/04-clusters/1-new-clusters/03-azure.md
Comment (codrin-iftimie) [inline content/docs/04-clusters/1-new-clusters/03-azure.md:104]: I'd enhance this note with the fact that you'd have to manually input the information for **Control plane Subnet** and **Worker Network**
```
Name <br /> CIDR Block <br /> Security Group Name
```
if these conditions are met

**Review by codrin-iftimie — CHANGES_REQUESTED** [review]

File: content/docs/04-clusters/1-new-clusters/03-azure.md
Comment (resmaa) [reply content/docs/04-clusters/1-new-clusters/03-azure.md:104]: done

File: content/docs/04-clusters/1-new-clusters/03-azure.md
Comment (codrin-iftimie) [inline content/docs/04-clusters/1-new-clusters/03-azure.md:102]: `<a href="">Control Plane Subnet</a>`

no need for this anchor

File: content/docs/04-clusters/1-new-clusters/04-aks.md
Comment (codrin-iftimie) [inline content/docs/04-clusters/1-new-clusters/04-aks.md:146]: `<a href="/glossary-all/#users">Worker Network</a>`

no need for this anchor

**Review by codrin-iftimie — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/877#pullrequestreview-1107003788" title="Approved">codrin-iftimie</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/877#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
